DH_Engine/Classes/DHVoiceChatRoom.uc

```diff
@@ -36,8 +43,42 @@ simulated event bool IsMember(PlayerReplicationInfo PRI, optional bool bNoCascad
             {
                 return MyPRI != none && MyPRI.SquadIndex == SquadIndex;
             }
+            else if (IsCommandChannel() && MyPRI.IsSquadLeader())
+            {
+                return true;
+            }
+            else if (!IsSquadChannel() && MyPRI.IsInSquad())
+            {
+                return false;
+            }
+            else if (IsPrivateChannel())
+            {
+                // Get owner pawn
+                if (PlayerReplicationInfo(Owner) != none && PlayerReplicationInfo(Owner).Owner != none && PlayerController(PlayerReplicationInfo(Owner).Owner).Pawn != none)
+                {
+                    OwnerPawn = PlayerController(PlayerReplicationInfo(Owner).Owner).Pawn;
+                }
+
+                // Get CheckPawn
+                if (MyPRI != none && MyPRI.Owner != none && PlayerController(MyPRI.Owner).Pawn != none)
+                {
+                    CheckPawn = PlayerController(MyPRI.Owner).Pawn;
+                }
 
-            return true;
+                if (OwnerPawn != none && CheckPawn != none && VSizeSquared(OwnerPawn.Location - CheckPawn.Location) < class'DHUnits'.static.MetersToUnreal(20))
```

Comment:
This is incorrect. You are calculated the *squared* distance between the two pawns, so the distance you're comparing it to needs to be squared as well, but you just have `class'DHUnits'.static.MetersToUnreal(20)`. I would probably also store the value somewhere so it doesn't need to be recalculated constantly (since this is a function that's going to be called very frequently).